### DISCUSSION TITLE
Crespo-Hernandez turbulence model

### DISCUSSION THREAD

**Discussion by cheoljoon** [NatLabRockies/floris#920] [Q&A] [answered]
Hi!

I have short questions about a turbulence model in FLORIS.
I noticed that there are two turbulence models in FLORIS: crespo_hernandez.py and wake_induced_mixing.py.
However, I also found that there are another wake-added turbulence models according to Renkema, 2007, such as 
Danish recommendation, Larsen, Frandsen, Quarton, etc. 
 
Is there any specific reason we prefer to use Crespo-Hernandez model to calculate wake-added turbulence intensity in FLORIS? 
Is there any reference that supports this reason? 
Is this because this turbulence model stood the test of time like Jensen's model? 

Thanks!

Reference: 
D. Renkema. Validation of wind turbine wake models. Master Thesis, 2007
A. Crespo and J. Hernanadez. Turbulence characteristics in wind-turbine wakes. Journal of wind engineering and industrial aerodynamics, 61(1): 71–85, 1996

**Comment by misi9170** [comment, marked as answer]
Hi @cheoljoon, sorry for the slow reply and thanks for the question!

The main reason that we mostly use the Crespo-Hernandez model is that the first key models in FLORIS were implemented partly according to [this paper by Niayifar and Porté-Agel](https://iopscience.iop.org/article/10.1088/1742-6596/625/1/012039), who use the turbulence model presented by [Crespo and Hernández](https://www.sciencedirect.com/science/article/pii/016761059500033X).

However, we have not looked deeply into wake added turbulence models; and, in all honesty, part of our reasoning for implementing the [`WakeInducedMixing` model](https://github.com/NREL/floris/blob/main/floris/core/wake_turbulence/wake_induced_mixing.py) that you also mentioned to work with the [Empirical Gaussian (EmG) wake model](https://nrel.github.io/floris/empirical_gauss_model.html#wake-induced-mixing) was to avoid a strict definition of "turbulence intensity" and focus instead on the key effect for wind turbine wakes, which is an increased mixing with the surrounding flow, using an empirical relationship that we could easily tune to match observations of wakes. 

We'd certainly be interested in studies that compare wake-added turbulence models, even (or possibly all the better) if those are models for other measures of mixing (such as turbulent kinetic energy) rather than turbulence intensity.

I'm sorry that this is not the most satisfying of answers, and again I'm sorry for the delayed response to your question!

Misha

**Reply by cjeong3** [reply]
Hi @misi9170, 

It helps me a lot to understand the underlying reason why FLROIS uses specific turbulence models. 
I will explore more in detail using the links you put in the text. 

Thank you very much!
